core/profiles/standard/standard.install

```diff
@@ -108,6 +108,42 @@ function standard_install() {
   theme_enable(array($default_theme));
   theme_disable(array('stark'));
 
+  // Enables the title_combo block and positions breadcrumbs
+  // immediately above title_combo block.
+
+  $title_combo_uuid = 'a549d0bc-51f7-46b0-9b41-653788410dd4';
+
```

Comment:
removing this line might also be nice? (optional)

Reply:
@jenlampton Done